### PR TITLE
Getting doxygen to run

### PR DESCRIPTION
**Pull request by cjnolet**

### CODE REVIEW

**Comment by cjnolet** [comment]
rerun tests

File: cpp/Doxyfile.in [hunk 1]
```diff
@@ -836,7 +828,9 @@ EXCLUDE_SYMLINKS       = NO
 # exclude all test directories for example use the pattern */test/*
 
 EXCLUDE_PATTERNS       = columnWiseSort.h \
-                         smoblocksolve.h
+                         smoblocksolve.h \
```
Comment (dantegd) [inline cpp/Doxyfile.in:831]: I think this and the above columnWiseSort are not needed?

**Review by dantegd — APPROVED** [review]
Just had a minor comment of a couple of excluded files that (if I'm not mistaken) are not part of the raft repo yet

File: cpp/Doxyfile.in (hunk 1 above)
Comment (cjnolet) [reply cpp/Doxyfile.in:831]: I think you are right. I'll remove those.

**Review by dantegd — APPROVED** [review]

**Comment by cjnolet** [comment]
@gpucibot merge

**Pull request merged**